PerfTools/JeProf/BuildFile.xml

```diff
@@ -0,0 +1,4 @@
+<use name="sockets"/>
```

Comment:
Why `sockets`?

Reply:
To get it to link to libdl.so. That is the only tool file that defines it. Otherwise I get an unknown symbols error for _dl_open iirc.

Reply:
@smuzaffar What would you recommend?

Reply:
@makortel , this is fine,  many tools (python3, libuuid etc.) also depend on `sockets`